modules/boilerplate/security.js

```diff
@@ -6,13 +6,7 @@ const csrfProtection = csrf({ cookie: true });
 
 // these URLs won't get the helmet header protection
 // @TODO this should only affect the legacy homepage
-const pathsExemptFromHelmet = [
-    '/',
-    '/welsh',
-    '/legacy',
-    '/funding/funding-finder',
-    '/welsh/funding/funding-finder'
-];
+const pathsExemptFromHelmet = ['/', '/welsh', '/legacy', '/funding/funding-finder', '/welsh/funding/funding-finder'];
```

Comment:
I'm not keen on this in general but when we switch off the various proxy routes we can kill this too.